modules/hcp-bm-hc.adoc

```diff
@@ -20,6 +20,8 @@ As you create a hosted cluster, keep the following guidelines in mind:
 
 - The most common service publishing strategy is to expose services through a load balancer. That strategy is the preferred method for exposing the Kubernetes API server. If you create a hosted cluster by using the web console or by using {rh-rhacm-title}, to set a publishing strategy for a service besides the Kubernetes API server, you must manually specify the `servicePublishingStrategy` information in the `HostedCluster` custom resource.
```

Comment:
We suggest to use a load balancer but by default with the hcp create agent command the bare metal hosted clusters are created with node port: we should indicate this here. We also should explain how to change the servicePublishingStrategy in the section to create bare metal hosted clusters as otherwise we leave this for the user to research with not even references to where. @jparrill ptal

Reply:
I updated the wording here, but I still need help to describe what users should do to change the servicePublishingStrategy.